srcpkgs/NetworkManager/template

```diff
@@ -47,6 +45,7 @@ make_dirs="
  /etc/NetworkManager/VPN 0755 root root
  /var/lib/NetworkManager 0755 root root
 "
+patch_args=-Np1
 
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
Please try to remove this block.

Reply:
Fixed in the most recent push.

Reply:
There might have been a misunderstanding? I wasn't talking about `patch_args` (that's fine, and you probably want to leave `-Np1`, which is the direction we want to move in, but rather about the `case XBPS_TARGET_MACHINE` block, which should no longer be necessary, maybe.

Reply:
The build succeeds even with that section removed.